tests/load/locustfile.py

```diff
@@ -42,578 +25,99 @@ def _(parser):
-        # should just be replaced with
-        # ```
-        # endpoint_url = res["pushEndpoint"]
-        # ```
-        if self.user.environment.parsed_options.endpoint_url and (
-            "dev" not in self.user.environment.parsed_options.websocket_url
-        ):
-            path = urlparse(res["pushEndpoint"]).path
-            endpoint_url = urljoin(
-                self.user.environment.parsed_options.endpoint_url, path
-            )
-        else:
-            endpoint_url = res["pushEndpoint"]
+    def disconnect(self) -> None:
+        self.ws.close()
 
-        # NOTE: Not sure why we're specifying a Topic here, but sure...?
-        self.headers.update({"Topic": "zyxw"})
-        while True:
-            # NOTE: This feels odd.
-            # We send a notification to the client, but then immediately
-            # drop the websocket connection. There's a small chance
-            # that the server already tried to deliver the message, but we
-            # are not ACK'ing it, so the server will simply retry on next
-            # connection. Why not drop the connection before we send?
-            endpoint_res = self.client.post(
-                url=endpoint_url,
-                name="ENDPOINT test_connect_forever",
-                data=self.encrypted_data,
-                headers=self.headers,
-            )
+    def hello(self) -> None:
+        with self._time_event(name="hello"):
+            body = json.dumps(dict(messageType="hello", use_webpush=True))
+            self.ws.send(body)
+            res = json.loads(self.ws.recv())
+            assert (
+                res["messageType"] == "hello"
+            ), f"Unexpected messageType. Expected: hello Actual: {res['messageType']}"
             assert (
-                endpoint_res.status_code == 201
-            ), f"status code was {endpoint_res.status_code}"
-            ws.close()
+                res["status"] == 200
+            ), f"Unexpected status. Expected: 200 Actual: {res['status']}"
+        self.uaid = res["uaid"]
 
-            # sit on our thumbs for 15 seconds.
-            time.sleep(15)
+    def register(self) -> None:
+        chid: str = str(uuid.uuid4())
 
-            # and then reconnect, but don't check if the response contains
-            # the previously sent Topic message, but send an Ack anyway? 🤨
-            ws = create_connection(
-                self.user.environment.parsed_options.websocket_url,
-                header={"Origin": "http://localhost:1337"},
-                ssl=False,
+        with self._time_event(name="register"):
+            body = json.dumps(dict(messageType="register", channelID=chid))
+            self.ws.send(body)
+            res = json.loads(self.ws.recv())
+            assert res["messageType"] == (
+                "register",
+                f"Unexpected messageType. Expected: register Actual: {res['messageType']}",
             )
```

Comment:
@jrbenny35 please apply this fix, Phil and I discovered it during calibration

```suggestion
            assert (
                 res["messageType"] == "register"
            ), f"Unexpected messageType. Expected: register Actual: {res['messageType']}"
```

Reply:
Looks like the linter messed this up huh

Reply:
I think I messed it up actually. sry about that